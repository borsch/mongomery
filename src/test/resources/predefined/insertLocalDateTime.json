{
  "TestCollection": [
    {
      "stringField": "some value",
      "localDateTimeField": "$insertLocalDateTime(2020-02-02T11:11:11)"
    }
  ]
}