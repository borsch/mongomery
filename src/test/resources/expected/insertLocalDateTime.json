{
  "TestCollection": [
    {
      "_id": "$anyObject()",
      "stringField": "some value",
      "localDateTimeField": "$eqLocalDateTimeValue(/2020-02-02T11:11:11/)"
    }
  ]
}